tests/foreman/ui/test_computeresource_rhev.py

```diff
@@ -98,6 +98,65 @@ def test_positive_create_rhev_with_name(self):
                     self.assertIsNotNone(self.compute_resource.search(name))
 
     @run_only_on('sat')
+    @skip_if_bug_open('bugzilla', 1467664)
+    @tier1
+    def test_positive_create_rhev_with_existing_name(self):
+        """Create a new rhev Compute Resource with existing name correction
+
+        :id: b9499d65-ef70-48b7-854a-f4cf740fbf9c
+
+        :setup: rhev hostname and credentials.
+
+        :steps:
+
+            1. Create a compute resource of type rhev.
+            2. Provide it with the valid hostname, username and password.
+            3. Provide a valid name to rhev Compute Resource.
+            4. Test the connection using Load Datacenter and submit.
+            5. Create again a compute resource with same name.
+            6. After error of name already taken,
+               update with different name
+            7. Click submit
+
+        :CaseAutomation: Automated
+
+        :expectedresults: A rhev CR is created successfully with proper
+            connection.
+
+        :CaseImportance: Critical
+
+        :BZ: 1467664
+        """
+        name = gen_string('alpha', 6)
+        new_name = gen_string('alpha', 6)
+        parameter_list = [
+            ['URL', self.rhev_url, 'field'],
+            ['Username', self.rhev_username, 'field'],
+            ['Password', self.rhev_password, 'field'],
+            ['Datacenter', self.rhev_datacenter, 'special select'],
+        ]
+        with Session(self.browser) as session:
+            make_resource(
+                session,
+                name=name,
+                provider_type=FOREMAN_PROVIDERS['rhev'],
+                parameter_list=parameter_list
+            )
+            self.assertIsNotNone(self.compute_resource.search(name))
+            make_resource(
+                session,
+                name=name,
+                provider_type=FOREMAN_PROVIDERS['rhev'],
+                parameter_list=parameter_list
+            )
+            self.compute_resource.assign_value(
```

Comment:
Ah, so user is redirected to 401 only after changing to valid name? My bad, misread BZ description. Then asserting "name already taken" error absence after hitting 'submit' button won't help us, as that error message is not present on 401-error page, that check will pass even if bz is not fixed. The easiest way is probably to check url to ensure we were not redirected. So my suggestion is:
```python
make_resource(...)
+ self.assertIsNotNone(
+     self.compute_resource.wait_until_element(common_locators["name_haserror"]))
self.compute_resource.assign_value(...)
self.compute_resource.click(...)
+ self.assertFalse(self.browser.current_url.endswith('katello/401'))
self.assertIsNotNone(self.compute_resource.search(new_name))
```